docker-compose/docker-compose.yaml

```diff
@@ -2,23 +2,75 @@ version: '2.4'
 services:
   # Description: Acts as a reverse proxy for all of the sourcegraph-frontend instances
   #
-  # Disk: none
+  # Disk: 1GB / persistent SSD
   # Ports exposed to other Sourcegraph services: none
-  # Ports exposed to the public internet: 7080 (HTTP) and/or 7443 (HTTPS)
-  #
-  # Note: nginx/nginx/sourcegraph_backend.conf lists all of the sourcegraph-frontend
-  # network addresses that Nginx should proxy. nginx/nginx/sourcegraph_backend.conf
-  # needs to be updated with new addresses when scaling the number of soucegraph-frontend
-  # replicas.
-  nginx:
-    container_name: nginx
-    image: 'index.docker.io/library/nginx:1.17.7@sha256:8aa7f6a9585d908a63e5e418dc5d14ae7467d2e36e1ab4f0d8f9d059a3d071ce'
+  # Ports exposed to the public internet: 80 (HTTP) and 443 (HTTPS)
+  #
+  # Notes:
```

Comment:
I suggest deleting all of these notes entirely, and the ones that are not self-explanatory or easily discovered be moved into actual documentation on docs.sourcegraph.com

- Do users really need to understand SRC_FRONTEND_ADDRESSES? Doesn't seem that important at face value to me, they can just accept what we provide or look at the code to see what it is doing.
- Instead of explaining what each of the Caddyfiles do, provide commented out configs or explain how to use these in a docs page on docs.sourcegraph.com if that sounds insufficient.